news/2020-11-27-project-loved-november-2020.md

```diff
@@ -0,0 +1,220 @@
+
+[![](/wiki/shared/news/2020-11-27-project-loved-november-2020/catch/4-chocolate.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[otetsu feat. GUMI - Chocolate](https://osu.ppy.sh/beatmapsets/36371#fruits) by [Kuzino](https://osu.ppy.sh/users/158552)\
+180 BPM, 2:02 | 5.28★\
+*description written by [Tenshichan](https://osu.ppy.sh/users/1101600)*
+
+Kuzino is certainly a name that comes to mind when you think about old mappers that made lots of good quality maps. The ones that have been Loved already show that the community definitely enjoys their mapping style and *Chocolate* is no exception to the rule. This is a map that does not require speed like most of today's maps, but rather tests your timing and catcher control. Due to the hyperdash change back in 2012, some of the jumps became more tricky than necessary because of the timing needed to catch them, but it is still very much possible to get a full combo. Both older and newer players can fully enjoy and experience this map, so make sure to give it a shot!
+
+## <a id="mania"></a>osu!mania
+
+osu!mania Loved candidates were chosen by [-mint-](https://osu.ppy.sh/users/8976576), [Abraxos](https://osu.ppy.sh/users/5025064), [lenpai](https://osu.ppy.sh/users/5314573), [Pope Gadget](https://osu.ppy.sh/users/2288341) and [Toaph Daddy](https://osu.ppy.sh/users/7616811)!
+
+---
+
+[![](/wiki/shared/news/2020-11-27-project-loved-november-2020/mania/1-shindeiru.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[Virtual Riot - Shindeiru](https://osu.ppy.sh/beatmapsets/866120#mania) by [Gekido-](https://osu.ppy.sh/users/4693052)\
+150 BPM, 2:34 | \[4K\] 4.67★
+
+Omae wa mo shindeiru. An iconic phrase that has beget many memes, jokes, and in this case, songs and charts. One such song is Virtual Riot's *Shindeiru*, charted by lemonguy (formerly Gekido-). lemonguy is no stranger to the Loved section, having been nominated for many popular and impressive tech and LN charts alike, such as [Chik Habit](https://osu.ppy.sh/beatmapsets/732982) and [The Clockwork Rose](https://osu.ppy.sh/beatmapsets/740979). This pattern continues with *Shindeiru*, an extremely technical and bursty file that will catch you off-guard if you aren't fully ready for the speed and difficulty of the patterns throughout. Outside of the bursts there are also some extended jack patterns that can be difficult to transition in and out of, especially with the one-handed trills that go along with them. It is a difficult and satisfying chart, but if you aren't up to the challenge, you may already be dead.
```

Comment:
"charted by lemonguy (formerly Gekido-)" but the line above says mapped by Gekido-

Reply:
Resolving by changing the description to say Gekido- instead of lemonguy